From 0ae0065586e27d433b40a456043461c9a70ce013 Mon Sep 17 00:00:00 2001
From: Shunqian Zheng <zhengsq@rock-chips.com>
Date: Tue, 11 Sep 2018 11:00:24 +0800
Subject: [PATCH] ARM: config: enable INPUT_RK8XX_PWRKEY for
 rockchip_linux_defconfig

CONFIG_INPUT_RK8XX_PWRKEY is required for pmic power key.

Change-Id: I7ca3e364f6b3beaebabbd7a79e4914e866414663
Signed-off-by: Shunqian Zheng <zhengsq@rock-chips.com>
---
 arch/arm/configs/rockchip_linux_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rockchip_linux_defconfig b/arch/arm/configs/rockchip_linux_defconfig
index 2e3d730734a7..f34123535b82 100644
--- a/arch/arm/configs/rockchip_linux_defconfig
+++ b/arch/arm/configs/rockchip_linux_defconfig
@@ -232,6 +232,7 @@ CONFIG_TOUCHSCREEN_GT9XX=y
 CONFIG_TOUCHSCREEN_ELAN=y
 CONFIG_TOUCHSCREEN_USB_COMPOSITE=y
 CONFIG_INPUT_MISC=y
+CONFIG_INPUT_RK8XX_PWRKEY=y
 CONFIG_INPUT_UINPUT=y
 CONFIG_INPUT_GPIO_ROTARY_ENCODER=y
 CONFIG_SERIO_RAW=y
-- 
2.35.3

